panther_driver/src/PantherKinematics.py

```diff
@@ -17,6 +17,7 @@ def __init__(self):
         self.robot_length = 0
         self.wheel_radius = 0  # Distance of the wheel center, to the roller center
         self.encoder_resolution = 0
+        self.gear_ratio = 0
```

Comment:
All attributes which are common for each panther should be defined as class attributes not instance attributes. I guess these ones are specifically attributes describing hardware values. (robot_width out of my mark scope, but the same case)

Reply:
I don't think there is a point of doing that. The code needs refactor.